clang/lib/Parse/ParseDecl.cpp

```diff
@@ -89,13 +89,23 @@ static StringRef normalizeAttrName(StringRef Name) {
   return Name;
 }
 
-/// isAttributeLateParsed - Return true if the attribute has arguments that
-/// require late parsing.
-static bool isAttributeLateParsed(const IdentifierInfo &II) {
+/// returns true iff attribute is annotated with `LateAttrParseExperimentalExt`
+/// in `Attr.td`.
+static bool IsAttributeLateParsedExperimentalExt(const IdentifierInfo &II) {
```

Comment:
Nit: should we make it consistent `IsAttributeLateParsedExperimentalExt` and `isAttributeLateParsedStandard`? One starts with capital I and the other starts with lower-case i.